src/sidebars/favourites/favourites.less

```diff
@@ -26,6 +26,10 @@
       background-color: @highlight-colour;
     }
   }
+
+  .darkens:hover {
+    background: #1F262E;
```

Comment:
Can we use a variable here? If one does not exist for this color, let's create one.